frontend_tests/zjsunit/zjquery.js

```diff
@@ -163,6 +163,16 @@ exports.make_new_elem = function (selector, opts) {
             event_store.generic_event('click', arg);
             return self;
         },
+        closest: function (selector) {
+            var elem = self;
+            var search = selector.startsWith('.') || selector.startsWith('#') ? selector.substring(1) : selector;
+            if (elem.selector.indexOf(search) > -1) {
```

Comment:
Can you add tests for this in the `zjquery` node test suite ?  I think that'd help me be confident on whether this has bugs (e.g. with one selector being a substring of the other).

Reply:
I've added a test for this. Thanks!